optaplanner-docs/src/main/asciidoc/Chapter-Optimization_algorithms/Chapter-Optimization_algorithms.adoc

```diff
@@ -258,13 +252,15 @@ Configure <<logging,logging>> to display the log messages of each scope.
 [[termination]]
 == Termination
 
-Not all phases terminate automatically and sometimes you don't want to wait that long anyway.
-A `Solver` can be terminated synchronously by up-front configuration or asynchronously from another thread.
+Not all phases terminate automatically and and may take too long.
```

Comment:
Just add a commit on your branch that fix this, it will automatically appear in this PR when you push to your fork :)